india_compliance/gst_india/doctype/gst_return_log/generate_gstr_1.py

```diff
@@ -623,7 +645,7 @@ def get_books_gstr1_data(self, filters, aggregate=False):
                 return books_data
 
         from_date, to_date = get_gstr_1_from_and_to_date(
-            filters.month_or_quarter, filters.year
+            filters.month_or_quarter, filters.year, filters.is_quarterly
```

Comment:
- Single month for first 2 months.
- 3 months for quarter ending
Only for books